run.go

```diff
@@ -22,25 +26,49 @@ func Run() error {
 		service.Registry.Register(service)
 	}
 
+	// Register probes for internal services
+	grpcStarted := RegisterHealthProbe("grpc", func(ctx context.Context) error {
```

Comment:
There's a "best practice" method of approach gRPC health checks, perhaps we should look into that. More info:
- https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-grpc-liveness-probe
- https://github.com/grpc/grpc/blob/master/doc/health-checking.md

Reply:
It's that available to us yet? Says v1.24 beta and we're v1.20 from the looks of it? Would be a good thing to use, less resource usage.